packages/firebase_auth/firebase_auth_web/lib/src/interop/auth_interop.dart

```diff
@@ -553,6 +557,7 @@ abstract class AuthError {
   external set tenantId(String s);
   external String get phoneNumber;
   external set phoneNumber(String s);
+  external dynamic get customData;
```

Comment:
minor nit: similarly here, consider using `Object` or `Object?` if you expect it to be nullable.

It is not ready yet, but in an upcoming release you'll be able to use a `JSObject` type instead, which will make it clear that this is not an arbitrary Dart object.